synth.py

```diff
@@ -11,7 +11,7 @@
   'README.md',
   'CONTRIBUTING.md',
   '.eslintignore',
-  '.eslintrc.yml',
+  '.eslintrc.json',
```

Comment:
are [these](https://github.com/googleapis/synthtool/tree/master/synthtool/gcp/templates/node_library) the templates? if so there's no longer a `.eslintrc.yml`

Reply:
I wonder if we removed it when we migrated to a newer version of gts at some point.